larpandoracontent/LArUtility/RANSAC/AbstractModel.h

```diff
@@ -0,0 +1,42 @@
+// Code is from https://github.com/drsrinathsridhar/GRANSAC
+// Original license file can be found in larpandoracontent/LArUtility/RANSAC/LICENSE.
```

Comment:
Location of these files is fairly random, so input welcomed.

I've included the original license since I did start off with some MIT licensed code...but its been pulled apart and been rewritten a few times, so 🤷 .